package/src/lib/octokit.ts

```diff
@@ -1,26 +1,34 @@
 import { Octokit } from "octokit";
-import type { OctokitResponse } from "@octokit/types";
+import type { Endpoints } from "@octokit/types";
 import { loadEnv } from "vite";
+import chalk from 'chalk';
 import pRretry from 'p-retry';
 import config from "virtual:astro-gists/config";
 
 // Load config options to check if verbose logging is enabled
 const isVerbose = config.verbose;
 
+// Define the GistResponse type
+type GistResponse = Endpoints["GET /gists/{gist_id}"]["response"];
+
 // Create Gist Logger interface
 const gistLogger = async (
   type: "info"|"warn"|"error", 
   message: string,
   VerboseCheck: boolean
   ) => {
+    // simplify console.log
+    const log = console.log;
+    // Create a log banner to identify logs from this package
+    const logBanner = chalk.blue("[")+chalk.blue("astro-gists : octokit")+chalk.blue("] ");
```

Comment:
Why can't this just be one string?

Reply:
0.0 i thought i fixed that... it used to be bolded on the brackets

Reply:
fixed